Makefile

```diff
@@ -103,11 +103,13 @@ ENV_MSG_CANT_GET = "Cannot get config vars for this service.  Check you are adde
 .env:
 	@if $(call IS_GIT_IGNORED,*.env*) && [ -e package.json ] && [ -z $(CIRCLECI) ]; then ($(call CONFIG_VARS,development,env) > .env && perl -pi -e 's/="(.*)"/=\1/' .env && $(DONE)) || (echo $(ENV_MSG_CANT_GET) && rm .env && exit 1); fi
 
+.pa11yci:
```

Comment:
Have a look at [line 99](https://github.com/Financial-Times/n-makefile/pull/120/files#diff-b67911656ef5d18c4ae36cb6741b7965R99), you should be able to add the file there right?